Doc/library/shlex.rst

```diff
@@ -49,11 +49,13 @@ The :mod:`shlex` module defines the following functions:
    .. versionadded:: 3.8
 
 
-.. function:: quote(s)
+.. function:: quote(s, always=False)
 
    Return a shell-escaped version of the string *s*.  The returned value is a
    string that can safely be used as one token in a shell command line, for
-   cases where you cannot use a list.
+   cases where you cannot use a list. If the *always* keyword argument is
+   set to ``True`` then the string *s* will always be escaped, even in the
+   absence of special characters.
```

Comment:
You should explain the purpose of the parameter: it's for performance, no?

Reply:
Addressed in 5f74da5d9ec1a24b2663364a974e70cb257016ed